ods_ci/tests/Tests/0600__distributed_workloads/test-run-distributed-workloads-tests_3.9.robot

```diff
@@ -0,0 +1,107 @@
+    [Documentation]    Run Go ODH test: TestKueueRayCpu
+    [Tags]  ODS-2514
+    ...     Tier1
+    ...     DistributedWorkloads
+    ...     Training
+    ...     WorkloadsOrchestration
+    Run DistributedWorkloads ODH Test    TestMnistRayCpu    ${RAY_CUDA_IMAGE_3.9}    ${NOTEBOOK_IMAGE_3.9}
+
+Run TestKueueRayCudaGpu ODH test with Python 3.9
+    [Documentation]    Run Go ODH test: TestKueueRayCudaGpu
+    [Tags]  Resources-GPU    NVIDIA-GPUs
+    ...     Tier1
+    ...     DistributedWorkloads
+    ...     Training
+    ...     WorkloadsOrchestration
+    Run DistributedWorkloads ODH Test    TestMnistRayCudaGpu    ${RAY_CUDA_IMAGE_3.9}    ${NOTEBOOK_IMAGE_3.9}
+
+Run TestKueueRayROCmGpu ODH test with Python 3.9
+    [Documentation]    Run Go ODH test: TestKueueRayROCmGpu
+    [Tags]  Resources-GPU    AMD-GPUs    ROCm
+    ...     Tier1
+    ...     DistributedWorkloads
+    ...     Training
+    ...     WorkloadsOrchestration
+    Run DistributedWorkloads ODH Test    TestMnistRayROCmGpu    ${RAY_ROCM_IMAGE_3.9}    ${NOTEBOOK_IMAGE_3.9}
+
+Run TestRayTuneHPOCpu ODH test with Python 3.9
+    [Documentation]    Run Go ODH test: TestMnistRayTuneHpoCpu
+    [Tags]  RHOAIENG-10004
+    ...     Tier1
+    ...     DistributedWorkloads
+    ...     Training
+    ...     WorkloadsOrchestration
+    Run DistributedWorkloads ODH Test    TestMnistRayTuneHpoCpu    ${RAY_CUDA_IMAGE_3.9}    ${NOTEBOOK_IMAGE_3.9}
+
+Run TestRayTuneHPOGpu ODH test with Python 3.9
+    [Documentation]    Run Go ODH test: TestMnistRayTuneHpoGpu
+    [Tags]  Resources-GPU    NVIDIA-GPUs
+    ...     Tier1
+    ...     DistributedWorkloads
+    ...     Training
+    ...     WorkloadsOrchestration
+    Run DistributedWorkloads ODH Test    TestMnistRayTuneHpoGpu    ${RAY_CUDA_IMAGE_3.9}    ${NOTEBOOK_IMAGE_3.9}
+
+Run TestKueueCustomRayCpu ODH test with Python 3.9
+    [Documentation]    Run Go ODH test: TestKueueCustomRayCpu
+    [Tags]  RHOAIENG-10013
+    ...     Tier1
+    ...     DistributedWorkloads
+    ...     Training
+    ...     WorkloadsOrchestration
+    Run DistributedWorkloads ODH Test    TestMnistCustomRayImageCpu    ${RAY_TORCH_CUDA_IMAGE_3.9}    ${NOTEBOOK_IMAGE_3.9}
+
+Run TestKueueCustomRayGpu ODH test with Python 3.9
+    [Documentation]    Run Go ODH test: TestKueueCustomRayGpu
+    [Tags]  RHOAIENG-10013
+    ...     Resources-GPU    NVIDIA-GPUs
+    ...     Tier1
+    ...     DistributedWorkloads
+    ...     Training
+    ...     WorkloadsOrchestration
+    Run DistributedWorkloads ODH Test    TestMnistCustomRayImageGpu    ${RAY_TORCH_CUDA_IMAGE_3.9}    ${NOTEBOOK_IMAGE_3.9}
+
+
+*** Keywords ***
+Prepare DistributedWorkloads Integration Test Suite for 3.9
+    [Documentation]    Prepare DistributedWorkloads Integration Test Suite for 3.9
+    # The bug fix for self signed certificate error is not available in codeflare-sdk version "v0.21.1" which is the
+    # last supported release for python 3.9 and hence skipping tests for self managed installtion
```

Comment:
we can add workaround of Oc login similar to [here](https://github.com/red-hat-data-services/ods-ci/blob/releases/2.14.0/ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource#L56) , if we want to run on self managed clusters too for self signed certificate error

Reply:
This workaround doesn't work here